### PR TITLE
Updating links to fix issue #2835

### PR DESCRIPTION
**Pull request by sdwheeler**
Fixing broken links and redirects. See #2835

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [x] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [x] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-3.0&branch=pr-en-us-2839) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-4.0&branch=pr-en-us-2839) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-5.0&branch=pr-en-us-2839) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-5.1&branch=pr-en-us-2839) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/6/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-6&branch=pr-en-us-2839) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/3.0/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/4.0/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/5.0/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/5.1/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/6/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/6/microsoft.powershell.security/set-authenticodesignature).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808211623082672-2839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7gSwcuwUxZHOzh8XDmbTWCe%2BMypLjzDg1UiuGNAWAlg%3D&st=2018-08-21T16%3A33%3A09Z&se=2018-09-21T16%3A38%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md [hunk 1]
```diff
@@ -151,7 +152,7 @@ Microsoft .NET Framework SDK (versions 1.1 and later) and in the Microsoft
 Windows SDK.
 
 For more information about the syntax and the parameter descriptions of the
-MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](http://go.microsoft.com/fwlink/?LinkId=119097)
+MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](/previous-versions/dotnet/netframework-2.0/bfsktky3(v=vs.80))
 in the MSDN library.
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md]: Remove reference to MSDN library now that the link points directly to Docs.

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ Microsoft .NET Framework SDK (versions 1.1 and later) and in the Microsoft
 Windows SDK.
 
 For more information about the syntax and the parameter descriptions of the
-MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](http://go.microsoft.com/fwlink/?LinkId=119097)
+MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](/previous-versions/dotnet/netframework-2.0/bfsktky3(v=vs.80))
 in the MSDN library.
```
Comment (DCtheGeek) [inline reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md]: Remove reference to MSDN library now that the link points directly to Docs.

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ Microsoft .NET Framework SDK (versions 1.1 and later) and in the Microsoft
 Windows SDK.
 
 For more information about the syntax and the parameter descriptions of the
-MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](http://go.microsoft.com/fwlink/?LinkId=119097)
+MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](/previous-versions/dotnet/netframework-2.0/bfsktky3(v=vs.80))
 in the MSDN library.
```
Comment (DCtheGeek) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md]: Remove reference to MSDN library now that the link points directly to Docs.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md [hunk 4]
```diff
@@ -152,7 +152,7 @@ Microsoft .NET Framework SDK (versions 1.1 and later) and in the Microsoft
 Windows SDK.
 
 For more information about the syntax and the parameter descriptions of the
-MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](http://go.microsoft.com/fwlink/?LinkId=119097)
+MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](/previous-versions/dotnet/netframework-2.0/bfsktky3(v=vs.80))
 in the MSDN library.
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md]: Remove reference to MSDN library now that the link points directly to Docs.

File: reference/6/Microsoft.PowerShell.Core/About/about_Signing.md [hunk 5]
```diff
@@ -152,7 +152,7 @@ Microsoft .NET Framework SDK (versions 1.1 and later) and in the Microsoft
 Windows SDK.
 
 For more information about the syntax and the parameter descriptions of the
-MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](http://go.microsoft.com/fwlink/?LinkId=119097)
+MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](/previous-versions/dotnet/netframework-2.0/bfsktky3(v=vs.80))
 in the MSDN library.
```
Comment (DCtheGeek) [inline reference/6/Microsoft.PowerShell.Core/About/about_Signing.md]: Remove reference to MSDN library now that the link points directly to Docs.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md]: Fixed

File: reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md (hunk 2 above)
Comment (sdwheeler) [reply reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md]: Fixed

File: reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md (hunk 3 above)
Comment (sdwheeler) [reply reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md]: Fixed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md (hunk 4 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md]: Fixed

File: reference/6/Microsoft.PowerShell.Core/About/about_Signing.md (hunk 5 above)
Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Core/About/about_Signing.md]: Fixed

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-3.0&branch=pr-en-us-2839) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-4.0&branch=pr-en-us-2839) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-5.0&branch=pr-en-us-2839) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-5.1&branch=pr-en-us-2839) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/6/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-6&branch=pr-en-us-2839) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/3.0/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/4.0/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/5.0/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/5.1/microsoft.powershell.security/set-authenticodesignature).

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/6/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.
 - **Line 110**: **[Warning]** Invalid file link:(~/6/microsoft.powershell.security/set-authenticodesignature).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808211711451686-2839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CWuV467XAUGAmDuXG3%2BaMTWwkWrVwgv7dcxi3Lbi73A%3D&st=2018-08-21T17%3A22%3A17Z&se=2018-09-21T17%3A27%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-3.0&branch=pr-en-us-2839) (powershell-3.0) | [Details](#user-content-reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-4.0&branch=pr-en-us-2839) (powershell-4.0) | [Details](#user-content-reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-5.0&branch=pr-en-us-2839) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-5.1&branch=pr-en-us-2839) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)
[reference/6/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/6/Microsoft.PowerShell.Core/About/about_Signing.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_signing?view=powershell-6&branch=pr-en-us-2839) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md)

<a name="reference%2F3.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F4.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/4.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.0/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/5.1/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Signing.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Signing.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue2835/reference/6/Microsoft.PowerShell.Core/About/about_Signing.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201808211739036152-2839%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zxN2DsHB8S2ozxggVKC0c6BpOOqr38HBhcoIKx1pXqw%3D&st=2018-08-21T17%3A50%3A15Z&se=2018-09-21T17%3A55%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**